### DISCUSSION TITLE
Listen to the recording

### DISCUSSION THREAD

**Discussion by edoconte** [mynaparrot/plugNmeet-server#47] [General]
i tried the demo and i would like to listen to the short recording i made but i don't know how to do it.

**Comment by jibon57** [comment]
You'll require to use [API](https://www.plugnmeet.org/docs/api/fetch_recording) to fetch the recordings. If you find it difficult then I'll recommend you to install any of [Wordpress](https://www.plugnmeet.org/docs/user-guide/wordPress-integration), [Moodle](https://www.plugnmeet.org/docs/user-guide/moodle-integration) or [Joomla](https://www.plugnmeet.org/docs/user-guide/joomla-integration) plugin in your website & use [demo API](https://www.plugnmeet.org/docs/intro#demo) for further testing. It will be much more flexible to use those plugins for joining sessions or downloading recordings.
